packages/core/src/generators/qwik/src-generator.ts

```diff
@@ -359,6 +359,10 @@ export class SrcBuilder {
         if (key === 'innerHTML') key = 'dangerouslySetInnerHTML';
         if (key === 'dataSet') return; // ignore
         if (self.isJSX) {
+          if (key.includes(':')) {
```

Comment:
To be safe, I would make sure the `value` is empty too:

```suggestion
          if (key.includes(':') && value == '') {
```

Reply:
Sure, thanks a lot! I've added it